packages/mockserver-config-writer/README.md

```diff
@@ -0,0 +1,33 @@
+# @sap-ux/mockserver-config-writer
+
+Adds or removes configuration for mockserver module `@sap-ux/ui5-middleware-fe-mockserver` to SAP UX project.
```

Comment:
`to SAP UX project` -> `to an SAP UX project`

Reply:
@kranthie-sap, I'll create a follow up for this.